libraries/app/database_api.cpp

```diff
@@ -2009,18 +2009,10 @@ set<public_key_type> database_api_impl::get_potential_signatures( const signed_t
             result.insert(k);
          return &auth;
       },
-      _db.get_global_properties().parameters.max_authority_depth
+		result,
+      _db.get_global_properties().parameters.max_authority_depth,
+		1
```

Comment:
We're still working on coding style guidelines. Try to format your code in a way that matches existing code.
It seems you're using tabs now instead of spaces. Please change that.

Reply:
already modified，please